openapi/components/schemas/KycRequestDocuments.yaml

```
@@ -0,0 +1,25 @@
+description: Documents to be requested from customer.
+type: array
+items:
+  type: object
+  required:
+    - type
+  properties:
+    type:
+      description: Document type to be requested from customer.
+      allOf:
+        - $ref: ./KycDocument/KycDocumentTypes.yaml
+    subtypes:
+      description: Document subtype allowed for customer.
+      type: array
+      items:
+        $ref: ./KycDocument/KycDocumentSubtypes.yaml
+    maxAttempts:
+      description: The number of document upload attempts. `0` treated as unlimited.
```

nitpick: based on the entity name, I'm assuming this is where the max number of attempts is set

```suggestion
      description: Total number of allowed document upload attempts. Use `0` to allow unlimited upload attempts.
```